### DISCUSSION TITLE
[Docs] 构建文档指定vitepress的base参数，增加一层根路径时，docs:build后，访问ip:port/my_root，页面打不开，显示Not Found

### DISCUSSION THREAD

**Discussion by QiChunDong** [element-plus/element-plus#17470] [Documentation]
Bug Type: **`Docs`**

## Environment

- Vue Version: `3.2.37`
- Element Plus Version: `2.7.6`
- Browser / OS: `Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:127.0) Gecko/20100101 Firefox/127.0`
- Build Tool: `Vite`

### Related Component

- `N/A`

### Reproduction Link

[Link]()

### Steps to reproduce

1. pnpm install 
2. pnpm run build
3. 修改`docs\.vitepress\config.mts`文件，增加base配置如下
```js
export const config: UserConfig = {
  title: 'Element Plus',
  description: 'A Vue 3 based component library for designers and developers',
  lastUpdated: true,
  base: '/ElementPlus/',
  head,
  ...
}
```
4. pnpm run docs:gen-locale
5. pnpm run docs:build
6. 通过nginx部署`docs\.vitepress\dist` 或者 `pnpm run docs:serve`，访问`http://localhost:port/ElementPlus/`，

### What is Expected?

正常打开文档页面，真唱本地离线访问

### What is actually happening?

页面重定向到了`http://localhost:5001/en-US/`, 响应报文看是`Not Found`

### Additional comments

_(empty)_

![图片](https://github.com/element-plus/element-plus/assets/28248796/38629edc-687c-49bc-905b-d61729e8f423)
![图片](https://github.com/element-plus/element-plus/assets/28248796/549c38b1-0490-47e3-a87b-e88315152f2a)

**Comment by DaXiong1** [comment]
请问解决了吗

**Comment by ethan897** [comment]
同问，解决了吗。。

**Reply by rzzf** [reply]
修改一下/docs/lang.js的重定向逻辑，然后剩下的就是nginx重定向的工作了
